cypress/integration/resources.spec.js

```diff
@@ -259,48 +260,38 @@ describe('resources', () => {
     cy.wait('@postLogin').its('status').should('eq', 200);
 
     cy.findByTestId(LOGIN_FORM).should('not.exist');
-  });
 
-  it('will allow user to upvote once logged in', () => {
     cy.findAllByTestId(UPVOTE_BUTTON)
       .first()
       .children('span')
       .invoke('text')
       .as('currentUpVoteCountText');
 
-    cy.server();
-    cy.route('PUT', 'api/v1/resources/**/upvote').as('upvote');
-
     cy.findAllByTestId(UPVOTE_BUTTON).first().click({ force: true });
     cy.wait('@upvote');
 
     cy.get('@upvote').then(({ status, response }) => {
       expect(status).to.eq(200);
       cy.get('@currentUpVoteCountText').should(
         'not.eq',
-        `Number of Number of upvotes:${response.body.resource.upvotes}`,
+        `YesNumber of upvotes:${response.body.resource.upvotes}`,
```

Comment:
I'm confused by this assertion and the one on line 294.

Why assert for content to not equal specific content after the API response returns. Shouldn't we know what the value is after an upvote/downvote and assert against that?

Reply:
I see what you are saying. I did this because there is no clean way of grabbing the current vote count.

If we wrap the actual count with a `span` or something and give it a test id, I guess we can assert for the content to be equal a response. What do you think?

https://github.com/OperationCode/front-end/blob/ca6998364c0aef1ecfd1cb7191ac5a948cba390e/components/Cards/ResourceCard/ResourceCard.js#L99

Reply:
Oh and also the each vote button would have to be clicked twice since depending on the vote status, the count could either increase/decrease by 1.

Reply:
We could wrap the number in a span and give it a testID to quickly grab the actual vote number.

Then we could findByTestId before and after the actual vote to ensure the count increments or decrements.

Reply:
Aren't that asserting for the value to not equal? Maybe I misunderstood what you were saying but I thought we were aiming at asserting for the value to be equal.

On a side note, I just noticed that `userVote` prop is not being passed into ResourceCard. If we had that information, it would definitely make things easier.

Reply:
You can essentially store the value by a variable and assert that after the PUT, that the value becomes -1 or +1